packages/x-tree-view/src/TreeItem2LabelInput/TreeItem2LabelInput.types.ts

```diff
@@ -1,8 +1,12 @@
-export interface TreeItem2LabelInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+import * as React from 'react';
+import { MuiCancellableEventHandler } from '../internals/models/MuiCancellableEvent';
+
+export interface TreeItem2LabelInputProps {
   value?: string;
-  onChange?: React.ChangeEventHandler<HTMLInputElement>;
-  /**
-   * Used to determine if the target of keydown or blur events is the input and prevent the event from propagating to the root.
-   */
```

Comment:
Maybe we should keep this?

Reply:
Indeed, I thought I could remove this prop at some point but since we are keeping it it should definitely have some JSDoc.